paddle/function/ConvOpTest.h

```diff
@@ -79,45 +79,54 @@ void Convolution(const std::string& conv1,
             if (outputChannels < inputChannels) continue;
             for (size_t stride : {1, 2}) {
               for (size_t padding : {0, 1}) {
-                if (padding >= filterSize) break;
+                for (size_t dilation : {1, 3}) {
+                  if (padding >= filterSize) break;
```

Comment:
Why here do not need some checking like in the 'Im2ColTest.cpp`?
`filterSize` multiplied by `dilation` may be greater than `inputSize`.
```
                  size_t filterSizeH = (filterHeight - 1) * dilation + 1;
                  size_t filterSizeW = (filterWidth - 1) * dilation + 1;
                  if (inputHeight <= filterSizeH || inputWidth <= filterSizeW)
                    break;
                  if (padding >= filterSizeH || padding >= filterSizeW) break;
                  size_t outputHeight =
                      (inputHeight - filterSizeH + 2 * padding) / stride + 1;
                  size_t outputWidth =
                      (inputWidth - filterSizeW + 2 * padding) / stride + 1;
```

Reply:
I add the control to it,  the `inputSize + 2 * padding < filterS`.